app/assets/stylesheets/components/_alert.scss

```diff
@@ -26,22 +25,22 @@ $ico-offset: -10px;
 
 .alert-notice,
 .alert-success {
-  background-color: #f0faed;
+  background-color: #ebfcef;
 
-  &::before { background-image: url(image-path('alert/ico-check.svg')); }
```

Comment:
I left it in there for now because it is being used by the progress bar, but we can remove it after that is gone.  Again, I will likely do an audit of all the various icons in the future to clean up an unnecessary images.